tools/influx_data_manager/influx_data_manager.xml

```diff
@@ -0,0 +1,88 @@
+<tool id="influx_data_manager" name="Influx Data Manager: Handling influx_si data inputs in Galaxy workflows" version="@TOOL_VERSION@+galaxy0" profile="23.0">
+    <macros>
+        <token name='@TOOL_VERSION@'>1.0.0</token>
+    </macros>
+    <requirements>
+        <requirement type='package' version='@TOOL_VERSION@'>influx-si-data-manager</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    influx_data_manager
+    --physiofit '$physiofit' 
+    --isocor '$isocor'
+    --linp '$linp'
+    --netw '$netw'
+    --mmet '$mmet'
+    --cnstr '$cnstr' 
+    --tvar '$tvar'
+    --opt '$opt'
+    --log '$stdout'
+    $verbose
+    2> '$stderr' && cat '$stdout'
+    ]]></command>
+    <inputs>
+        <param type='data' format='tabular,txt' argument='--netw' label='netw' help='File containing network for experiments (.netw file)' optional='false'/>
+        <param type='data' format='tabular,txt' argument='--isocor' label='miso' help='Output of isocor which corresponds to the .miso input' optional='false'/>
+        <param type='data' format='tabular,txt' argument='--tvar' label='tvar' help='File containing types of variables for experiments (.tvar file)' optional='false'/>
+        <param type='data' format='tabular,txt' argument='--physiofit' label='mflux' help='Output of physiofit which corresponds to the .mflux input' optional='true'/>
+        <param type='data' format='tabular,txt' argument='--linp' label='linp' help='File contianing label inputs for experiments (.linp file)' optional='true'/>
+        <param type='data' format='tabular,txt' argument='--mmet' label='mmet' help='File containing specie concentrations for experiments (.mmet file)' optional='true'/>
+        <param type='data' format='tabular,txt' argument='--cnstr' label='cnstr' help='File containing constraints onf fluxes and specie concentrations for experiments (.constr file)' optional='true'/>
+        <param type='data' format='tabular,txt' argument='--opt' label='opt' help='File containing options for influx_si (.opt  file)' optional='true'/>
+        <param name='verbose' type='boolean' truevalue='--verbose' falsevalue='' label='Debug mode' help='Select "Yes" to have debug logs be printed out'/>
+    </inputs>
+    <outputs>
+        <data format='txt' name='stdout' label='Run log'/>
```

Comment:
Just a note, Galaxy is handling the stdout and stderr on its own. There is no need to add them here.

Reply:
Yes I will change how this is handled in the next tool update